app/models/job.rb

```diff
@@ -63,10 +63,10 @@ def check_active_on_destroy
     true
   end
 
-  def self.agent_state_update_queue(jobid, state, message = nil)
+  def self.agent_message_update_queue(jobid, message)
```

Comment:
thank you @kbrock , I missed that we would need migration otherwise since there could be messages in the queue